okteto/okteto
# add depot.dev as a valid builder

## Proposed changes
- add new `isDepotEnabled()` method to check if depot builder is enabled by checking needed env vars: `OKTETO_DEPOT_TOKEN` and `OKTETO_DEPOT_PROJECT_ID`. This builder will be used always in case both envs are set
- refactor `getSolveOpt` method to reuse code for `buildWithOkteto` and `buildWithDepot` builders.
- add integration test for buildWithDepot fn

## How to validate changes
- check that a build using okteto works as expected
- set `OKTETO_DEPOT_TOKEN` and `OKTETO_DEPOT_PROJECT_ID` envs:
- run `okteto build` and check image is build and push to the target registry.
